src/main/python/__init__.py

```diff
@@ -151,14 +151,15 @@ def _ppr_single_target(self, edge_type, seeds, target, relative_error=0.1, reset
 
 
     # TempestDB methods
-    def ppr(self, edge_type, seeds, num_steps=100000, reset_probability=0.3, max_results = None):
+    def ppr(self, edge_type, seeds, seed_node_type, target_node_type, num_steps=100000, reset_probability=0.3, alternating = True, max_results = None):
```

Comment:
splitting into two calls works. We should talk about the semantics of the two calls (Eg, what do you mean by SALSA: is it simply a walk on a bipartite graph, or is an alternating walk on a directed graph, or something else?)